Dolphin scripts/Entrance Randomizer/__main__.py

```diff
@@ -37,6 +38,7 @@
     state,
 )
 
+remove_disabled_exits()
```

Comment:
this is now called in `set_transitions_map`, and makes copies of the original `ALL_POSSIBLE_TRANSITIONS` and `TRANSITION_INFOS_DICT` instead of modifying them directly.